submissionaccesses.md

```diff
@@ -0,0 +1,48 @@
+# SubmissionAccesses Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/config/submissionaccesses**   
```

Comment:
I would prefer to rename this endpoint to be either `/submissionaccessoptions` or `/submissionpermissions`. I feel the name `/submissionaccesses`, is confusing, as it sounds like it relates to statistics (as "accesses" is more often used to represent the number of times an object has been accessed/viewed... and it doesn't imply that you are modifying access permissions).